src/patterns/ObjectTile/ObjectTile.tsx

```diff
@@ -0,0 +1,174 @@
+import { ComponentProps, ElementType, forwardRef } from 'react';
+import classnames from 'classnames';
+
+import { getCommonProps } from '../../utils';
+import { Text, TextVariants } from '../../components/Text';
+import { DetailList } from '../DetailList/index';
+import { Detail } from '../../components/Detail/index';
+import { PhillipsLogo } from '../../assets/icons';
+
+export interface ObjectTileProps extends ComponentProps<'a'> {
+  /**
+   * Optional Object badge.
+   */
+  badgeText?: string;
+  /**
+   * Optional Element to render at the top level.
+   */
+  element?: ElementType<
+    Omit<ObjectTileProps, 'imageUrl' | 'lotNumber' | 'referenceNumber'> & { 'data-testid': string }
+  >;
+  /**
+   * Estimate for object.
+   */
+  estimate?: string;
+  /**
+   * Estimate label text.
+   */
+  estimateLabelText?: string;
+  /**
+   * Image URL for the object.
+   */
+  imageUrl?: string;
+  /**
+   * Object Lot number.
+   */
+  lotNumber: string;
+  /**
+   * Object Maker name.
+   */
+  makerText?: string;
+  /**
+   * Object Model text.
+   */
+  modelText?: string;
+  /**
+   * Object reference number.
+   */
+  referenceNumber?: string;
+  /**
+   * Object title.
+   */
+  titleText?: string;
+  /**
+   * Withdrawn Text
+   */
+  withdrawnText?: string;
+}
+/**
+ * ## Overview
+ *
+ * The ObjectTile component is used to display an object tile with an image, title, and other information in a grid
+ *
+ * [Figma Link](https://www.figma.com/design/hMu9IWH5N3KamJy8tLFdyV/EASEL-Compendium%3A-Tokens%2C-Components-%26-Patterns?node-id=10914-34792&node-type=canvas&m=dev)
+ *
+ * [Storybook Link](https://phillips-seldon.netlify.app//?path=/story/patterns-objecttile--playground)
+ */
+const ObjectTile = forwardRef<HTMLAnchorElement, ObjectTileProps>(
+  (
+    {
+      badgeText,
+      className,
+      children,
+      element: Element,
+      estimate,
+      estimateLabelText = 'Estimate',
+      imageUrl,
+      lotNumber,
+      makerText,
+      modelText,
+      referenceNumber,
+      titleText,
+      withdrawnText,
+      ...props
+    },
+    ref,
+  ) => {
+    const { className: baseClassName, ...commonProps } = getCommonProps(props, 'ObjectTile');
+    Element = (props.href ? 'a' : 'div') as ElementType<
```

Comment:
will `element` prop ever get used if passed in? seems like it can only ever be `a` or `div`. maybe this should be `props.href ? 'a' : Element` and `Element` defaults to `div`